### ISSUE TITLE
[Feature Request] Animation Bank

### ISSUE THREAD

**Issue by yerarto** [ThemeFuse/Brizy#476]
In the demo version there are 4 types of animation I think. They are great, but It would be awsome to have more different types with more deep customization with the possibility to apply them to any element (backgrounds fade in, smooth scrolling applied to images,  :)

**Comment by vitog** [comment]
Agreed. I would like Slider Revolution integration. Currently when I upload it, in the template, Brizy Build becomes narrow. @DimiBaitanciuc

**Comment by CarlosA8** [comment]
Are they adding more animations?

**Comment by CarlosA8** [comment]
if not, how can you add the css animations for it?
